src/main/avro/GuessService.avdl

```diff
@@ -0,0 +1,27 @@
+@version("1.0")
+@namespace("com.github.rakhmedovrs.avro")
+protocol GuessService {
+
+	enum RequestIntention {
```

Comment:
Perhaps having this as RequestType or MessageType with
CONNECT
GUESS
DISCONNECT

could be more straightforward here.

Reply:
Fixed